specs/light_client/merkle_proofs.md

```diff
@@ -61,25 +96,27 @@ We can now define a concept of a "path", a way of describing a function that tak
 ```python
 def item_length(typ: SSZType) -> int:
     """
-    Returns the number of bytes in a basic type, or 32 (a full hash) for compound types.
+    Return the number of bytes in a basic type, or 32 (a full hash) for compound types.
     """
     if issubclass(typ, BasicValue):
         return typ.byte_len
     else:
         return 32
-        
-        
-def get_elem_type(typ: ComplexType, index: Union[int, str]) -> Type:
+```
+
+```python
+def get_elem_type(typ: Union[BaseList, Container], index_or_variable_name: Union[int, SSZVariableName]) -> SSZType:
```

Comment:
@vbuterin @protolambda 
`ComplexType` has not been defined yet. I use `Union[BaseList, Container]` to replace it now, could you confirm it's what you expected?

Reply:
It must be one of `Bytes`, `BytesN`, `List`, `Vector`, `Container`. Does `BaseList` cover all of the former four?

Reply:
Thank you for the elaboration! `BaseList` only covers `List` and `Vector`. I added `BaseList` to cover `Bytes` and `BytesN` in https://github.com/ethereum/eth2.0-specs/pull/1363/commits/a1061f0b2f777b8c2e55d8134edf95885a5607f6.